build/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM quay.io/operator-framework/ansible-operator:v0.12.0
 
-COPY watches.yaml ${HOME}/watches.yaml
+RUN pip3 install --user jmespath
```

Comment:
This permits using the `json_query()` Ansible filter to get the elastic default user password.